src/frontend/apps/markdown/MdxRenderer/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import * as runtime from 'react/jsx-runtime.js';
+
+import { Box } from 'grommet';
+import { Nullable } from 'utils/types';
+import { Spinner } from 'components/Loader';
+import { debounce } from 'lodash';
+
+import { evaluate, EvaluateOptions } from '@mdx-js/mdx2';
```

Comment:
The original dependency name is `mdx-js/mdx` but the test stack uses a former version of it (version 1.x), so I decided to add the dependency with a different name to avoid conflicts...